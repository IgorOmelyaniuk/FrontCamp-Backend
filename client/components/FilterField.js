import React from 'react'

const FilterField = (props) => {
    return (
        <div className="form-group col-3">
            <label>Filter blogs by author:</label>
            <input type="text" className="form-control" value={props.author} onChange={props.filterByAuthor} />
        </div>
    )
}

export default FilterField