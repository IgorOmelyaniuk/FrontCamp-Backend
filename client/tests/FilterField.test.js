import React, { Component } from 'react';
import { shallow } from 'enzyme';
import FilterField from '../components/FilterField'; 
import toJson from 'enzyme-to-json'

describe('subscribe component', () => {
  it('should render FilterField', () => {
    expect(toJson(shallow(<FilterField/>))).toMatchSnapshot();
  });
});