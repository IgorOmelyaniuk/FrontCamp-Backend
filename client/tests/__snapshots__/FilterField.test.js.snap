// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`subscribe component should render FilterField 1`] = `
<div
  className="form-group col-5"
>
  <label>
    Filter blogs by author:
  </label>
  <input
    className="form-control"
    type="text"
  />
</div>
`;
